securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -307,6 +304,7 @@ Options is an Object, where:
 * checkLoginIframeInterval - Set the interval to check login state (default is 5 seconds).
 * responseMode - Set the OpenID Connect response mode send to {project_name} server at login request. Valid values are query or fragment . Default value is fragment, which means that after successful authentication will {project_name} redirect to javascript application with OpenID Connect parameters added in URL fragment. This is generally safer and recommended over query.
 * flow - Set the OpenID Connect flow. Valid values are standard, implicit or hybrid.
+* promiseType - If set to `native` all methods returning a promise will return a native JavaScript promise. If not set will return Keycloak specific promise objects.
```

Comment:
"If not set will return" sounds weird to me, but I let it on @matthewhelmke to check this.

Reply:
The phrasing is okay since it is parallel to the first sentence, "If set to ... will return..."

Reply:
Ok, my mistake, thanks